collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+- Install relevant CLIs for your provider of choice:
+  - AWS: [`aws` cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [`ejsctk`](https://eksctl.io/introduction/#installation)
+  - Azure: [`az` cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  - GCP: [`gcloud` cli](https://cloud.google.com/sdk/gcloud)
+
+### Create your cluster
+- Follow the instructions for your provider of choice to create your Kubernetes (k8s) cluster: [Cloud installation instructions](https://docs.sourcegraph.com/admin/deploy/kubernetes#cloud-installation)
+  - Follow the recommendations for `node type`, `disk type`, and `minimum` and `maximum` nodes
+
+### Configure your manifest file
+Sourcegraph has a list of configurations available and recommended that need to be made to your manifest file to properly install Sourcegraph into your cluster. Please review the full configuration guide prior to proceeding: [Configure Sourcegraph with Kubernetes](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure)
+
+Get started by following these steps to fork the [`deploy-sourcegraph`](https://github.com/sourcegraph/deploy-sourcegraph) repository: [Getting started](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#getting-started)
+
+- Configurations:
+  - [License Key](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#add-license-key)
+  - [Storage Class](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#configure-a-storage-class)
+  - [Network Access](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#configure-network-access)
+  - [External PostgreSQL Database](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#sourcegraph-databases)
+  - [Scaling for services](https://docs.sourcegraph.com/admin/deploy/kubernetes/scale#tuning-replica-counts-for-horizontal-scalability)
+  - [Cluster role administrator access](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+
+### Install Sourcegraph
+With your manifest file configured, ensure that you can [access your cluster](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/) with `kubectl` and `kubectl` is pointed to the proper context.
+
+- Follow the [Direct installation](https://docs.sourcegraph.com/admin/deploy/kubernetes#direct-installation) steps to apply your manifest to your cluster
+- Once your deployment is completed, you can access Sourcegraph via the [ingress](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#ingress-controller-recommended) you configured in your manifest
+
+## Next steps
+Congratulations, Sourcegraph is now deployed and you can begin to configure your instance. We recommend you review the [Configurating Sourcegraph](https://docs.sourcegraph.com/admin/config) documentation next to [add a code host](https://docs.sourcegraph.com/admin/external_service), [configure SSO](https://docs.sourcegraph.com/admin/auth), and more.
```

Comment:
```suggestion
Congratulations, Sourcegraph is now deployed and you can begin to configure your instance. We recommend you review the [Configurating Sourcegraph](https://docs.sourcegraph.com/admin/config) documentation next to [add a code host](https://docs.sourcegraph.com/admin/external_service), [configure authN and authZ](https://docs.sourcegraph.com/admin/auth), and more.
```
May want to consider linking to https://docs.sourcegraph.com/admin/config/authorization_and_authentication instead, since it's oriented more towards "if you want x do y" than the main auth docs.

Reply:
Before I accept this @emchap, can you add the suggestion with the other auth piece? That makes sense to me if that's used and referenced more often than the main auth documentation. I think you're saying to do it this way?

```suggestion
Congratulations, Sourcegraph is now deployed and you can begin to configure your instance. We recommend you review the [Configurating Sourcegraph](https://docs.sourcegraph.com/admin/config) documentation next to [add a code host](https://docs.sourcegraph.com/admin/external_service), [configure authN and authZ](https://docs.sourcegraph.com/admin/config/authorization_and_authentication), and more.
```
The original link has a lot of auth information though on it, is it really not useful and shouldn't be here? Maybe both?

Reply:
@emchap that looks great to me!